File: 백준/Bronze/3059. 등장하지 않는 문자의 합/등장하지 않는 문자의 합.c
#include <stdio.h>
#include <string.h>

int main() {
	int n;
	char str[1000];
	int alphabetChecker[26];
	int asciiSum;

	scanf("%d", &n);

	for (int i = 0; i < n; i++) {
		asciiSum = 0;

		for (int i = 0; i < 26; i++) {
			alphabetChecker[i] = 0;
		}

		scanf("%s", str);

		for (int i = 0; i < strlen(str); i++) {
			alphabetChecker[str[i] - 'A'] = 1;
		}

		for (int i = 0; i < 26; i++) {
			if (alphabetChecker[i] == 0) {
				asciiSum = asciiSum + (i + 'A');
			}
		}

		printf("%d\n", asciiSum);
	}

	return 0;
}

File: 백준/Bronze/3059. 등장하지 않는 문자의 합/등장하지 않는 문자의 합.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n;
        String str;
        int[] alphabetChecker = new int[26];
        int asciiSum;

        n = sc.nextInt();
        sc.nextLine();

        for (int i = 0; i < n; i++) {
            asciiSum = 0;

            for (int j = 0; j < 26; j++) {
                alphabetChecker[j] = 0;
            }

            str = sc.nextLine();

            for (int j = 0; j < str.length(); j++) {
                alphabetChecker[str.charAt(j) - 'A'] = 1;
            }

            for (int j = 0; j < 26; j++) {
                if (alphabetChecker[j] == 0) {
                    asciiSum = asciiSum + (j + 'A');
                }
            }

            System.out.println(asciiSum);

        }
        sc.close();
    }
}
